indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -248,6 +252,12 @@
             $checkbox.prop('checked', !$checkbox.prop('checked')).trigger('change');
             toggleResetBtn();
         });
+
+        var $eventParticipantsList = $('#event-participants-list');
```

Comment:
since you use it only once, no need to assign it first. btw, let's use `const` in new code, even when adding it in an old file